src/System.Reflection/tests/AssemblyNameTests.cs

```diff
@@ -430,7 +428,6 @@ public static void Constructor_String_VersionTest(Version expectedVersion, strin
         }
 
         [Fact]
-        [ActiveIssue("https://github.com/dotnet/corert/issues/4321", TargetFrameworkMonikers.UapAot)]
```

Comment:
Can these issues be closed now?

Reply:
I just closed 4321. 2113 is still only partly addressed - method invocations still need toolchain work to be enabled for pointers.